kieker-c-build/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:jammy
+
+run apt-get update -y ; apt-get install -y gcc make autotools-dev autoconf automake libtool libltdl7 libltdl-dev binutils-dev build-essential devscripts debhelper libcmocka-dev libcmocka0
```

Comment:
I'm not sure whether it has a big impact, but maybe add an apt-get clean at the end to keep the image as small as possible.